tests/test_guarantee_msg_order.py

```diff
@@ -9,7 +9,7 @@ def test_mismatch(server_guarantee_msg_order):
 
     # We use an exception handle to catch errors raised by the server
     def handle_exception(loop, context):
-        msg = context.get("exception", context["message"])
+        msg = str(context.get("exception", context["message"]))
```

Comment:
Should we change how `context["exception"]` and `context["message"]` are set to ensure they are `stress?

Reply:
I am not sure if that is possible. It is defined here: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.set_exception_handler

Reply:
Oh in that case wouldn't they already be `str`s?

Reply:
`context["exception"]` is a `ValueError`. I use `str()` to assert the message of the error.